src/data/quality-assurance-quiz.ts

```diff
@@ -1,4 +1,142 @@
+    Answer: "Static testing",
+    Distractor1: "Dynamic testing",
+    Distractor2: "Black box testing",
+    Distractor3: "White box testing",
+    Explanation:
+      "Static testing focuses on examining the source code and project documents to catch defects early in the software development life cycle.",
+    Link: "https://www.altexsoft.com/whitepapers/quality-assurance-quality-control-and-testing-the-basics-of-software-quality-management/"
+  },
+
+  {
+    Question:
+      "What software testing principle suggests that defects tend to cluster in certain areas of the software?",
+    Answer: "Defect clustering.",
+    Distractor1: "Testing shows the presence of mistakes.",
+    Distractor2: "Exhaustive testing is impossible.",
+    Distractor3: "Testing is context dependent.",
+    Explanation:
+      "Defect clustering indicates that approximately 80 percent of defects are found in only 20 percent of the system modules, making it essential to test thoroughly in those areas.",
+    Link: "https://www.altexsoft.com/whitepapers/quality-assurance-quality-control-and-testing-the-basics-of-software-quality-management/"
+  },
+  {
+    Question: "What is the primary goal of software testing?",
+    Answer: "To identify and correct defects in the software.",
+    Distractor1: "To speed up the software development process.",
+    Distractor2: "To create documentation for the software.",
+    Distractor3: "To design the user interface of the software.",
+    Explanation:
+      "Software testing aims to identify and correct defects in the software to ensure its quality and reliability.",
+    Link: "https://www.altexsoft.com/whitepapers/quality-assurance-quality-control-and-testing-the-basics-of-software-quality-management/"
+  },
+
+  {
+    Question:
+      "What is the main purpose of a test strategy in software testing?",
+    Answer: "To provide a roadmap for the testing process.",
+    Distractor1: "To list all the defects in the software.",
+    Distractor2: "To define the user requirements.",
+    Distractor3: "To create a test plan.",
+    Explanation:
+      "A test strategy serves to provide a roadmap that describes the steps to be conducted in testing and how much effort and resources are required.",
+    Link: "https://www.altexsoft.com/whitepapers/quality-assurance-quality-control-and-testing-the-basics-of-software-quality-management/"
+  },
+
+  {
+    Question:
+      "What type of testing focuses on verifying the software's compliance with functional requirements?",
+    Answer: "Black box testing",
+    Distractor1: "White box testing",
+    Distractor2: "Grey box testing",
+    Distractor3: "Ad hoc testing",
+    Explanation:
+      "Black box testing focuses on testing the software's functionality without looking at its internal code or structure.",
+    Link: "https://www.altexsoft.com/whitepapers/quality-assurance-quality-control-and-testing-the-basics-of-software-quality-management/"
```

Comment:
we only accept links from freeCodeCamp resources, or wikipedia.
please change this